rustfmt.toml

```diff
@@ -1,3 +1,4 @@
 error_on_line_overflow = true
 error_on_unformatted = true
 version = "Two"
+format_cargo_toml = true
```

Comment:
Same comment as above. I think we should hold off on applying this to rustfmt until after we've added this feature.